sql/core/src/test/resources/sql-tests/results/literals.sql.out

```diff
@@ -353,10 +353,12 @@ pattern%	no-pattern\%	pattern\%	pattern\\%
 select '\'', '"', '\n', '\r', '\t', 'Z'
 -- !query schema
 struct<':string,":string,
-:string,
:string,	:string,Z:string>
+:string,
+:string,	:string,Z:string>
 -- !query output
 '	"	
-	
			Z
```

Comment:
https://github.com/apache/spark/blob/ec8952747d38c9510a040b1f95d636444359088f/sql/core/src/test/resources/sql-tests/results/literals.sql.out#L352-L359

@MaxGekk Do you know why there is a diff here? I execute git diff locally without this part

Reply:
fiixed